src/base/AEmbeddedRanking.ts

```diff
@@ -105,6 +106,14 @@ export abstract class AEmbeddedRanking<T extends IRow> implements IViewProvider
       runWithoutTracking<T>(f: () => T): Promise<T> {
         return super.withoutTracking(f);
       }
+
+      protected getParameterFormDescs() {
```

Comment:
```suggestion
      protected getParameterFormDescs(): IFormElementDesc[] {
```

I would type the return value.